spec/no_yast2_packages.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -x
+
+# Check if AutoYaST2 packages were installed.
```

Comment:
I guess that you wanted to

  Check that AutoYast packages were not installed

;)